src/handler.ts

```diff
@@ -6,8 +6,8 @@ import { LogLevel } from "./log-level";
  * Alternatively custom handlers can extend the abstract class `BaseHandler`.
  */
 export interface ILogHandler {
-  readonly level: LogLevel;
-  handle(record: ILogRecord): void;
+  readonly level: LogLevel | LogLevel[];
+  handle(record: ILogRecord | LogLevel[]): void;
```

Comment:
This is bullshit. Handle should not accept an array of loglevels. It was late in the evening. Must have been slipped through...